src/Presentation/Salutations.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Presentation;
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Exception\ValidatorException;
+use Symfony\Component\Validator\Validation;
+
+class Salutations {
+
+	/**
+	 * @var array
+	 */
+	private array $salutations;
+
+	public function __construct( array $salutations ) {
+		if ( !$this->isValid( $salutations ) ) {
```

Comment:
This will validate the salutations on almost every request. I think in the case of salutations, where the value does not change often, but is heavily used, this is a bad tradeoff between security and performance, as negligible as the performance cost of this validation might be. Could you move the validity check to the frontend-content-repo and instead describe the array shape of the constructor parameter? 

 I like your use of Symfony Validators